tests/integration/testdata/sync/039-partials/kong-partials-no-name.yaml

```diff
@@ -0,0 +1,6 @@
+_format_version: "3.0"
```

Comment:
If partial can be created without providing name/id, how does it get linked to a plugin 🤔

Reply:
In that case, a user would have to take a gateway dump and use those ids.
Can be useful in distributed config.